src/lib/schematics/ng-add/index.spec.ts

```diff
@@ -142,4 +154,28 @@ describe('ng-add schematic', () => {
         'Expected the project main file to not contain a HammerJS import.');
     });
   });
+
+  describe('animations disabled', () => {
+
+    it('should not add @angular/animations to package.json', () => {
```

Comment:
Isn't this _always_ required, though, since `@angular/material` imports stuff from `@angular/animations`?

Reply:
I think it depends from where people import. Most imports from `@angular/animations` are just types that won't be part of the JS.
 
My reasoning for this option was just that there might be people who don't want the schematics to touch their `package.json`/`app.module.ts`. For the general public, I think it would be better if we just add the `NoopAnimationsModule` or `BrowserAnimationsModule` and always ensure `@angular/animations` is set up?

Reply:
It's probably worth checking, but I'm reasonable sure that the app won't build if `@angular/animations` isn't installed. I don't want there to be any paths where the user ends up with an app that doesn't build.

Reply:
Yeah, it would be wrong to open any "path" where the user ends up with an application that doesn't build.

I've updated this PR to either add the `NoopAnimationsModule` or `BrowserAnimationsModule`. This also includes a safety check that ensures that we don't "magically" add the `BrowserAnimationsModule` or `NoopAnimationsModule` if the project already explicitly uses an animation module. Please have another look.